bin/docker_entrypoint.sh

```diff
@@ -1,7 +1,10 @@
-#!/bin/bash
+#!/usr/bin/bash
```

Comment:
Is this used anywhere else - want to make sure that its parameterized if so

Reply:
![image](https://github.com/chroma-core/chroma/assets/126764/c7c7a903-f59a-4b21-9860-bb264cff531a)

@HammadB it's the default path for the base image that is used in Dockerfile. Are you sure that requires customization?

Reply:
@MrZoidberg, you'll find that running the base `python:3.10-slim-bookworm` image with `/bin/bash` works as fine. I think the reason the path needs changing is because of the added `bash` package in the `Dockerfile` above.

Reply:
reverted to /bin/bash